taxcalc/parameters.py

```diff
@@ -6,7 +6,7 @@
 
 DEFAULT_START_YEAR = 2013
 
-class Parameters(object):
+class Params(object):
```

Comment:
I'm fine with calling the attribute on the `Calculator` object `params` instead of `parameters`, but I think it's OK to leave the class name as `Parameters` as one is rarely typing out the full class name. Just a thought.